magentoerpconnect/product_category.py

```diff
@@ -265,7 +250,7 @@ def parent_id(self, record):
         mag_cat_id = binder.to_openerp(record['parent_id'])
 
         if category_id is None:
-            raise MappingError("The product category with "
-                               "magento id %s is not imported." %
+            raise MappingError(_("The product category with "
+                                 "magento id %s is not imported.") %
```

Comment:
What I mean by technical errors is that they are not supposed to be shown to the users with a `UserError` popup.

I distinguish "error messages" with are meant entirely for the users and should be localized and "exceptions messages" that must not be translated. These exceptions messages are for developers and not end-users. This is important to be able to search for errors and in some case to analyze the content of the exception in `try: except:`.